### ISSUE TITLE
Add notes about multiple bindings for the same keys + mods + mode.

### ISSUE THREAD

**Issue by kchibisov** [alacritty/alacritty#3273]
Our documentation doesn't mention that you can have things like that

```yml
- { key: Y,        mods: Control, action: ScrollLineUp,   mode: ~Alt       }
- { key: Y,        mods: Control, action: ScrollLineUp,   mode: ~Alt       }
- { key: Y,        mods: Control, action: ScrollLineUp,   mode: ~Alt       }
- { key: Y,        mods: Control, action: ScrollLineUp,   mode: ~Alt       }
- { key: Y,        mods: Control, action: ScrollLineUp,   mode: ~Alt       }
```

to for e.g. scroll 5 lines up on `Ctrl + Y`. This thing could be very useful if you want to accelerate scrolling or some other bindings. Also note that we can have different actions but the same `key + mods + mode`, so the following will scroll only one line up.

```yml
   - { key: I,        mods: Alt,     action: ScrollLineUp,   mode: ~Alt       }
   - { key: I,        mods: Alt,     action: ScrollLineUp,   mode: ~Alt       }
   - { key: I,        mods: Alt,     action: ScrollLineDown, mode: ~Alt       }
```

So we should add notes about such things (but word a bit nicer than I did here) to our `alacritty.yml`, since it's the source of our documentation.

**Comment by Brainface1** [comment]
How about something like
`Note that the same trigger can be assigned to multiple actions. All of the actions are then executed at once.`
Maybe that last part isn't necessary.

**Comment by chrisduerr** [comment]
I'd say the last part is crucial. But generally that would suffice, yeah. Just might rephrase it a bit, since `Note that...` usually shouldn't be necessary.

**Comment by chrisduerr** [comment]
Yeah, that sounds better.

**Comment by Brainface1** [comment]
I'd put it directly before the `#key_bindings` line, as the part before that is the only thing that is related to the behaviour of triggers.

**Comment by chrisduerr** [comment]
@Brainface1 Feel free to send a PR, we can discuss further changes in there then. That makes it a bit easier because we can make use of Github's review process and direct code annotations and suggestions.

**Comment by Brainface1** [comment]
Just created one: #3296

**Comment by kchibisov** [comment]
closed via #3296
